locales/en.default.json

```diff
@@ -64,7 +64,8 @@
       "checkout": "Check Out",
       "empty": "Your cart is currently empty.",
       "continue_browsing_html": "Continue browsing <a href=\"/collections/all\">here</a>.",
-      "close_cart": "Close Cart"
+      "close_cart": "Close Cart",
+      "savings": "You're saving"
```

Comment:
Should we add the price right in this string? Some other languages might prefer the price and then savings text?